app/src/features/sessionBook/screens/SavedSessionScreen/components/SavedSessionViewer/SavedSessionViewer.tsx

```diff
@@ -72,8 +72,7 @@ export const SavedSessionViewer = () => {
             <DownloadSessionButton />
           </div>
         </div>
-        {/* TODO: Add bottom sheet */}
-        <BottomSheetLayout bottomSheet={<>Session Bottom sheet here</>}>
+        <BottomSheetLayout bottomSheet={<SessionViewerBottomSheet playerTimeOffset={sessionPlayerOffset} />}>
```

Comment:
Are we sending any analytics event for bottomsheet?

Reply:
BottomSheet itself has some analytics events, and I've also added tab clicked events in this PR: 
https://github.com/requestly/requestly/pull/1969